src/window.rs

```diff
@@ -182,6 +182,11 @@ impl Window {
         self.window.set_inner_size(size);
     }
 
+    // TODO: use `with_position` once available
+    pub fn set_position(&mut self, x: i32, y: i32) {
+        self.window.set_position(LogicalPosition::from((x, y)));
```

Comment:
The x/y coordinates in the config and CLI are in physical units, to get the expected behavior for the user we need to convert them to logical units using the DPR. So you should be creating a `PhysicalPosition` with `x` and `y` here and then using `dpr` to convert it to a `LogicalPosition` (there's a method on `PhysicalPosition` to do that I believe).